src/main/java/net/sourceforge/jaad/mp4/MP4InputStream.java

```diff
@@ -420,8 +420,19 @@ public void skipBytes(final long n) throws IOException {
 		}
 
 		while(l<n) {
-			if(in!=null) l += in.skip((n-l));
-			else if(fin!=null) l += fin.skipBytes((int) (n-l));
+			if(in!=null) {
+				l += in.skip((n-l));
+			}
+			else {
+				if(fin!=null) {
+					if ((n - l) > Integer.MAX_VALUE) {
```

Comment:
Shouldn't this be a while loop? What if n-l is twice or more as much as Integer.MAX_VALUE?